ckanext/syndicate/tasks.py

```diff
@@ -139,9 +167,28 @@ def _create_package(package):
         set_syndicated_id(package, remote_package['id'])
     except toolkit.ValidationError as e:
         if 'That URL is already in use.' in e.error_dict.get('name', []):
-            logger.info("package with name '{0}' already exists".format(
+            logger.info("package with name '{0}' already exists. Check creator.".format(
                 new_package_data['name']))
-        raise e
+            author = get_syndicated_author()
+            if author is None:
+                raise
+            try:
+                remote_package = ckan.action.package_show(id=new_package_data['name'])
+                remote_user = ckan.action.user_show(id=author)
+            except Exception:
```

Comment:
Again we are catching a general exception here (either package_show or user_show failed). It would be good to know which call failed and log a more useful message